roles/openshift_health_checker/library/search_journalctl.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
+"""Interface to journalctl"""
+
+from time import time
+from datetime import datetime
+import json
+import re
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+class InvalidMatcherRegexpException(Exception):
+    """Base exception class for invalid matcher regexp fields"""
+    def __init__(self, message):
+        Exception.__init__(self, message)
+
+
+def stamp_too_old(stamp_microseconds, stamp_limit_seconds):
+    """Receives a log message time stamp (in microseconds) and determines if it
```

Comment:
For all method/function docstrings in this PR: please change the verb tense to imperative. And please add punctuation where missing.

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

https://www.python.org/dev/peps/pep-0257/#one-line-docstrings